docs/users.rst

```diff
@@ -759,3 +759,13 @@ releases.
 
 
 .. _`config.json`: https://github.com/containerbuildsystem/atomic-reactor/blob/master/atomic_reactor/schemas/config.json
+
+With enabled  :ref:`OMPS integration <omps-integration>` operator manifests are
```

Comment:
This ref is not processed correctly in github view, I'm not sure how to fix it

Reply:
It does render properly when building html docs though. Should not be a problem at all.